docs/development.rst

```diff
@@ -5,6 +5,53 @@
 
 This document contains information on developing llbuild.
 
+Build Instructions
+------------------ 
+
+**Building from source on OSX**
+
+* Install latest Xcode and dependencies::
+
+    $ brew install cmake ninja
+
+* Install FileCheck
+
+* Build::
+
+    $ mkdir build && cd build
+    $ cmake -G Ninja -DCMAKE_BUILD_TYPE:=Debug ..
+    $ ninja
+
+* If cmake errors out with : `Failed to locate 'lit' executable (missing: LIT_EXECUTABLE)`::
```

Comment:
This will only work if the user has installed `lit` locally. We should probably mention that part.

Also, I think there is a way to get the path explicitly, I believe I do it in the Xcode config which automatically installs `lit`.